src/conditional-flow/multiple-conditions.js

```diff
@@ -1,19 +1,15 @@
 // This function should return true if num is greater
 // than or equal to lower AND less than or equal to upper.
 // Implement this with a single condition.
-function isInRange (num, lower, upper) {
-
-  // TODO: write code in this function body to pass the tests
-
+function isInRange(num, lower, upper) {
+  return num >= lower && num <= upper ? true : false;
```

Comment:
You can just return `num >= lower && num <= upper` here as that will create a true or false value. No need  to use the ternary operator

Reply:
I'll try to remember that for next time